docs/user_guide/trace.md

```diff
@@ -81,10 +81,10 @@ The following table shows available global trace settings to pass to `--trace-co
     <td>OFF</td>
     <td>
       Indicates the level of trace detail that should be collected and <br/>
-      may be specified  multiple times to trace multiple informations. <br/>
+      may be specified  multiple times to trace multiple information. <br/>
```

Comment:
Same here. Let me know if this needs to be rephrased

Reply:
Nice catch! Yeah, that could be good.